hibernate-core/src/main/java/org/hibernate/engine/jdbc/internal/StatementPreparerImpl.java

```diff
@@ -37,6 +38,11 @@
 	 */
 	StatementPreparerImpl(JdbcCoordinatorImpl jdbcCoordinator) {
 		this.jdbcCoordinator = jdbcCoordinator;
+		this.jdbcServices = jdbcCoordinator
```

Comment:
this still looks quite "heavy" . I see `JdbcCoordinatorImpl` is the only place invoking this ctor, and it also retrives the `jdbcServices` in its own constructor.. keep it and pass it on directly?

Reply:
I'll do